### PR TITLE
Added mobile navigation and made some changes

### PR DESCRIPTION
**Pull request by Signior-X**

### CODE REVIEW

**Review by pseudosodium — CHANGES_REQUESTED** [review]
In the mobile navigation change the list item names and their hyperlinks accordingly

File: main/templates/main/includes/navbar.html [hunk 1]
```diff
@@ -13,3 +16,19 @@
...
   </div>
 </nav>
+
+<!-- Mobile Navigation -->
+<ul class="sidenav" id="mobile-demo">
+  <li><a href="sass.html">Sass</a></li>
+  <li><a href="badges.html">Components</a></li>
+  <li><a href="collapsible.html">Javascript</a></li>
+  <li><a href="mobile.html">Mobile</a></li>
+</ul>
```
Comment (pseudosodium) [inline main/templates/main/includes/navbar.html]: Change these accordingly

**Review by pseudosodium — CHANGES_REQUESTED** [review]

File: main/templates/main/includes/navbar.html (hunk 1 above)
Comment (Signior-X) [reply main/templates/main/includes/navbar.html]: Arre yes, yeh maine kya kiya

**Comment by Signior-X** [comment]
check now, it is done.

File: main/templates/main/includes/navbar.html (hunk 1 above)
Comment (pseudosodium) [reply main/templates/main/includes/navbar.html]: I've made the changes accordingly... thanks for your contribution.
I'm closing this request.

**Comment by pseudosodium** [comment]
Changes have been made now with few other additions. New features would be welcomed and also raising issues would help a lot to make it better.

**Pull request closed without merging**